app.js

```diff
@@ -5,13 +5,16 @@ const fs = require('fs')
 const path = require('path')
 const port = (process.env.PORT || 3000)
 const yaml = require('js-yaml')
-const helperFunctions = require('./lib/helper-functions.js')
-const dto = require('./lib/dto-helper.js')
+
+const helperFunctions = require('./lib/helper-functions.js')  // misc helpers
+const dto = require('./lib/dto-helper.js')                    // directoryToObject helper
```

Comment:
Can we avoid single character variables?

E.g. `const directoryToObject = require('./lib/helpers/directory-to-object.js')`

Reply:
Sounds good, but we'll end up having `directoryToObject.directoryToObject()`